### DISCUSSION TITLE
Reinstall always fail

### DISCUSSION THREAD

**Discussion by y0d4a** [Security-Onion-Solutions/securityonion#13007] [Unsupported Versions]
Hi, i am struggling to reinstall SO with "so-setup-network"
Whatever i do, he always fail. 
Log say:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/3115997/cee1dd7c-4822-4e3f-988d-dc27b217b019)

**Comment by dougburks** [comment]
Please note that network installations are not officially supported:
https://docs.securityonion.net/en/2.4/network-installation.html

Please try a fresh installation of the latest ISO image:
https://docs.securityonion.net/en/2.4/installation.html

Make sure that you are following the hardware requirements and Best Practices:
https://docs.securityonion.net/en/2.4/hardware.html
https://docs.securityonion.net/en/2.4/best-practices.html
